app-i18n/librime/librime-1.9.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2012-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit cmake
+
+DESCRIPTION="RIME (Rime Input Method Engine) core library"
+HOMEPAGE="https://rime.im/ https://github.com/rime/librime"
+SRC_URI="https://github.com/rime/librime/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/1-${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-cpp/glog:=
+	>=dev-libs/boost-1.74:=
+	app-i18n/opencc:0=
+	dev-cpp/yaml-cpp:0=
+	dev-libs/leveldb:0=
+	dev-libs/marisa:0=
+"
+DEPEND="${RDEPEND}
+	dev-libs/darts
+	dev-libs/utfcpp
+	test? ( dev-cpp/gtest )
+"
+BDEPEND=""
+
+DOCS=(CHANGELOG.md README.md)
+
+src_configure() {
+	local -x CXXFLAGS="${CXXFLAGS} -I${ESYSROOT}/usr/include/utf8cpp"
```

Comment:
Can we address what this is for?

Reply:
This is for the utfcpp library which is not included in 1.7.3, however with 1.9.0, the utfcpp and darts headers are included in the package it self, so I will remove it and above DEPEND, thanks for this good debug!

Reply:
Can we make it build against system headers?
That is `dev-libs/darts` `dev-libs/utfcpp` `x11-libs/xcb-util-keysyms`.
If not, we should include LICENSE for corresponding headers.

Reply:
I tend to use the included header files, also I will append additional licenses.